tests/test_acasclient.py

```diff
@@ -2544,6 +2546,24 @@ def test_051_bulk_load_update_parent_alias(self):
         self.assertIn(alias_two, aliases)
         self.assertIn(alias_three, aliases) 
 
+        # Redo the same file (copy) to see if the result is the same 
+        # and aliases aren't just being appended without comparison 
+        test_051_upload_file_file_one_copy = Path(__file__).resolve().parent\
+            .joinpath('test_acasclient', 'test_051_register_parent_aliases_copy.sdf')
+        files = self.client.upload_files([test_051_upload_file_file_one_copy])
+        request["fileName"] = files['files'][0]["name"]
+        response = self.client.register_sdf_request(request)
+        self.assertIn('reportFiles', response[0])
+        self.assertIn('summary', response[0])
+        self.assertIn('Number of entries processed', response[0]['summary'])
```

Comment:
I know it's not your primary purpose in this PR, but could you add a check for how many compounds vs lots of new compounds were registered? This could function as a test of registering subsequent lots, which is something we're missing and would've helped me catch the bug I recently introduced.

Reply:
Yep, I'll add that!

Reply:
I've changed my mind - I think my request here is unnecessary scope creep. I can add a separate, simple test for registering subsequent lots. There's enough going on in this test already.